resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/domain/OrphanedChildJsonMixin.java

```diff
@@ -1,30 +1,30 @@
-/*
- * Copyright 2019 Crown Copyright
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package uk.gov.gchq.palisade.service.palisade.service;
-
-import uk.gov.gchq.palisade.service.Service;
-import uk.gov.gchq.palisade.service.palisade.exception.NoCapacityException;
-import uk.gov.gchq.palisade.service.palisade.request.ReadRequest;
-import uk.gov.gchq.palisade.service.palisade.request.ReadResponse;
-
-import java.util.concurrent.CompletableFuture;
-
-public interface DataService extends Service {
-
-    CompletableFuture<ReadResponse> read(final ReadRequest request) throws NoCapacityException;
-
-}
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service.resource.domain;

```

Comment:
Note that this has nothing to do with the DataService interface, git has just gone mad